docs/source/sysadmin_guide/systemd.rst

```diff
@@ -0,0 +1,85 @@
+*****************************
+Running Sawtooth As A Service
+*****************************
+
+When installing Sawtooth using apt-get, *systemd* units are added for the
+following components. These can then be started, stopped, and restarted using
+the *systemctl* command:
+
+* validator
+* transaction processors
+* rest_api
+
+To learn more about *systemd* and the *systemctl* command, check out `this
+guide`_:
+
+.. _this guide: https://www.digitalocean.com/community/tutorials/how-to-use-systemctl-to-manage-systemd-services-and-units
+
+
+Viewing Console Output
+----------------------
+
+To view the console output that you would see if you ran the components
+manually, run the following command:
+
+.. code-block:: console
+
+  $ sudo journalctl -f \
+  -u sawtooth-validator \
+  -u sawtooth-tp_config \
+  -u sawtooth-tp_validator_registry \
+  -u sawtooth-rest_api
+
+Validator Start-up Process
+==========================
+
+Create Genesis Block
+--------------------
+
+The first validator created in a new network must load a genesis block on
+creation to enable other validators to join the network. Prior to starting the
+first validator, run the following commands to generate a genesis block that
+the first validator can load:
+
+.. code-block:: console
+
+  $ sudo -u sawtooth -s
+  $ cd /tmp
+  $ sawtooth keygen --key-dir /tmp
+  $ sawtooth config genesis --key /tmp/sawtooth.priv
+  $ sawtooth admin genesis config-genesis.batch
+
+Running Sawtooth
+----------------
+
+To start a component using *systemd*, run following command where `component`
```

Comment:
Might be clearer to use COMPONENT so that the examples below are 'sawtooth-COMPONENT', making it clearer what string should be replaced.